docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/unified/index.md

```diff
@@ -0,0 +1,273 @@
+| `entities_or_sdes`   | A list of dictionaries defining the `context` or `self-describing` event tables to join onto your base events table. Please use the tool below or see the section on [Utilizing custom contexts or SDEs](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-utils-data-model/dbt-utils-advanced-operation/index.md?warehouse=redshift%2Bpostgres#utilizing-custom-contexts-or-sdes) for details of the structure | `[]`    |
+
+</TabItem>
+<TabItem value="bigquery" label="Bigquery" default>
+
+| Variable Name                | Description                                                                                         | Default |
+| ---------------------------- | --------------------------------------------------------------------------------------------------- | ------- |
+| `derived_tstamp_partitioned` | Boolean to enable filtering the events table on `derived_tstamp` in addition to `collector_tstamp`. | `true`  |
+
+</TabItem>
+<TabItem value="snowflake" label="Snowflake" default>
+
+| Variable Name | Description                                                                                                                                                                      | Default        |
+| ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- |
+| `query_tag`   | This sets the value of the `query_tag` for all sql executed against the database. This is used internally for metric gathering in Snowflake and its value should not be changed. | `snowplow_dbt` |
+
+</TabItem>
+</Tabs>
+
+## Output Schemas
+```mdx-code-block
+import DbtSchemas from "@site/docs/reusable/dbt-schemas/_index.md";
+import CodeBlock from '@theme/CodeBlock';
+import { SchemaSetterWSeeds } from '@site/src/components/DbtSchemaSelector';
+
+<DbtSchemas/>
+
+export const printSchemaVariables = (manifestSchema, scratchSchema, derivedSchema, seedSchema) => {
+  return(
+    <>
+    <CodeBlock language="yaml">
+    {`models:
+  snowplow_unified:
+    base:
+      manifest:
+        +schema: ${manifestSchema}
+      scratch:
+        +schema: ${scratchSchema}
+    sessions:
+      +schema: ${derivedSchema}
+      scratch:
+        +schema: ${scratchSchema}
+    user_mapping:
+      +schema: ${derivedSchema}
+    users:
+      +schema: ${derivedSchema}
+      scratch:
+        +schema: ${scratchSchema}
+    page_views:
+      +schema: ${derivedSchema}
+      scratch:
+        +schema: ${scratchSchema}`}
+        </CodeBlock>
+    </>
+  )
+}
```

Comment:
You need to add 
seeds:
  snowplow_web:
    +schema: ${seedSchema}`}
 if you want it here